Explorer/Assets/Scripts/CrdtEcsBridge/RestrictedActions/RestrictedActionsAPIImplementation.cs

```diff
@@ -74,15 +74,39 @@ public void TeleportTo(Vector2Int coords)
             TeleportAsync(coords).Forget();
         }
 
-        public bool ChangeRealm(string realm)
+        public bool ChangeRealm(string message, string realm)
         {
             if (!sceneStateProvider.IsCurrent)
             {
                 ReportHub.LogError(ReportCategory.RESTRICTED_ACTIONS, "ChangeRealm: Player is not inside of scene");
                 return false;
             }
 
-            ChangeRealmAsync(realm).Forget();
+            ChangeRealmAsync(message, realm).Forget();
+            return true;
+        }
+
+        public void TriggerEmote(string predefinedEmote)
+        {
+            if (!sceneStateProvider.IsCurrent)
+            {
+                ReportHub.LogError(ReportCategory.RESTRICTED_ACTIONS, "TriggerEmote: Player is not inside of scene");
+                return;
+            }
+
+            // TODO: Implement emote triggering (blocked until emotes are implemented)...
```

Comment:
@Kinerius are you sure? I think it would be better to leave it already merged into `main` so we will avoid future conflicts. In addition, this PR also includes some fixes for the `changeRealm` action that would be nice to have it available for the upcoming tests. The fact of having `triggerEmote`/`triggerSceneEmote` features incompleted is totally safe, it's not gonna affect to anything in our system.

Anyway, let me know when do you estimate that we're gonna have the avatar's emotes feature implemented. If it's gonna be short, I wouldn't mind to keep this PR open in the meanwhile, but to be honest I don't see it necessary.

Reply:
I agree any reason @Kinerius we cannot add these stubs for now?

Reply:
Fair enough